azerothcore/azerothcore-wotlk
# fix(DB/Spawning/Pathing) Ghostlands Respawn and path three areas

## Changes Proposed:
-  Respawn and path three areas in Ghostlands.

## Issues Addressed:
- Closes #7479

## SOURCE:
Sniff

## Tests Performed:
- Compiles
- Paths are correct
- Spawning now looks like Official

## How to Test the Changes:
1. .go c 81810 to see Nerubis Guard
2. .go c 81785 to see Anok'suten
3. .go c 82035 to see Lieutenant Tomathren
4. .go c 81854 to see Shadowpine
5. .go c 81871 to see Ravening Apparition
